CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ Read more about the session argument for computed fields in the [docs](https://h
 ### Bug fixes and improvements
 
 - console: make GraphiQL Explorer taking the whole viewport
+- cli: add insecure-skip-tls-verification and certificate-authority flags (close #4564) (#4582)
```

Comment:
This is just a minor clean-up. Could you move all the cli entries below console entries?

Reply:
I am adding a detailed changelog for this.